idd/Energy+.idd.in

```diff
@@ -44025,7 +44025,13 @@ ZoneHVAC:LowTemperatureRadiant:VariableFlow,
         \key SurfaceFaceTemperature
         \key SurfaceInteriorTemperature
         \default MeanAirTemperature
-   A6 , \field Heating Design Capacity Method
+   A6,  \field Setpoint Control Type
+        \note (How setpoint temperature is defined)
```

Comment:
I would take out the parentheses.  The use of a `\note` sorta implies that this is just a footnote of added info anyway.

Reply:
They are gone now.

Reply:
:+1: